charts/business-central/templates/load-balancer.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.service.sshEnabled }}
```

Comment:
thinking about it again we can just check if `.Values.service.ports.ssh` is set.